src/app/Shared/Services/Api.service.tsx

```diff
@@ -1486,7 +1488,12 @@ export class ApiService {
       } else {
         Promise.resolve(error.xmlHttpResponse.body as string).then((detail) => {
           if (!suppressNotifications) {
-            this.notifications.danger(`Request failed (${error.xmlHttpResponse.status} ${error.message})`, detail);
+            try {
+              const body = JSON.parse(detail).data.reason;
+              this.notifications.danger(`${title} Upload Failed`, body);
```

Comment:
I suppose these failures won't strictly be because of "Upload Failed". It would probably be better to just do ex. `this.notifications.danger(title, body)` here, and append "Upload Failed" to each case where that is relevant. Right now that might happen to be every case, but it doesn't seem to necessarily apply in this very generic error handling function.